rfcs/_process.md

```diff
@@ -0,0 +1,144 @@
+
+- Fill in the RFC.
+  Put care into the details: RFCs that do not present convincing motivation, demonstrate lack of understanding of the design's impact, or are disingenuous about the drawbacks or alternatives tend to be poorly received.
+
+- Submit a pull request.
+  As a pull request the RFC will receive design feedback from the various stakeholders, and the author should be prepared to revise it in response.
+
+- If your RFC is about something that is convenient to show using code, for example a sample of how new tests might be written if the RFC is accepted, you may find it useful to open an illustrative pull request alongside the RFC pull request.
+  Mark this illustrative pull request as a draft and make sure it has a link to the RFC.
+
+- Build consensus and integrate feedback.
+  RFCs that have broad support are much more likely to make progress than those that don't receive any comments.
+  Feel free to reach out to the maintainers group to get help identifying stakeholders and obstacles.
+
+- The stakeholders will discuss the RFC pull request, as much as possible in the comment thread of the pull request itself.
+  Offline discussion will be summarized on the pull request comment thread.
+
+- RFCs rarely go through this process unchanged, especially as alternatives and drawbacks are shown.
+  You can make edits, big and small, to the RFC to clarify or change the design, but make changes as new commits to the pull request, and leave a comment on the pull request explaining your changes.
+  Specifically, do not squash or rebase commits after they are visible on the pull request.
+
+- At some point, a maintainer will propose a "motion for final comment period" (FCP), along with a disposition for the RFC (merge, close, or postpone).
+  - This step is taken when enough of the tradeoffs have been discussed that the maintainers group is in a position to make a decision.
+    That does not require consensus among all participants in the RFC thread (which is usually impossible).
+    However, the argument supporting the disposition on the RFC needs to have already been clearly articulated.
+    Maintainers use their best judgment in taking this step, and the FCP itself ensures there is ample time and notification for stakeholders to push back if it is made prematurely.
+  - For RFCs with lengthy discussion, the motion to FCP is usually preceded by a summary comment trying to lay out the current state of the discussion and major tradeoffs/points of disagreement.
+  - Before actually entering FCP, the maintainers group must sign off as a whole.
+    This requires the usual "consensus minus one".
+
+- The FCP lasts 10 calendar days, so that it is open for at least 5 business days, absent holidays.
+  It is also advertised widely, e.g. on the discussion forum and in the [TC39 General](https://matrix.to/#/#tc39-general:matrix.org) Matrix channel.
```

Comment:
We should be more specific about where to publish it. I advertised it in TC39 General and TC39 Implementers on Matrix.